### ISSUE TITLE
Still supported in 2021?

### ISSUE THREAD

**Issue by khjeldnes** [laurenth-personal/lightmap-switching-tool#35]
Hi! 
So, this tool looked promising, and seemed to do what it should. However, when I bake the 2nd and 3rd lighting scene in your example map, then unity will remove the existing lightmap data for the static scene, and move the reference to the last baked lighting scene. So, I end up with lighting 1 and 2 pointing to data located in lighting 3.   So, when I rebake your example scene, I get Lightmap-1_xxx only in the last bake (lighting 3)  Is this a new behavour from unity that breaks your script? 
I also get a warning on "Store", where it's recommended to instantiate a scriptable object instead of creating  a new.

Do you have any solutions? Thanks!
Kjetil

**Comment by laurenth-personal** [comment]
Hello,
Thank you for reporting these issues.
I have indeed not tested the tool actively on 2021 and 2022 releases of
unity so I am not sure if anything changed.
There used to be a line in the script explicitly there to remove the
reference to the lighting data from the previous bake in order to avoid
unity detecting its existence and delete it.
Maybe I broke that in an update.
I can run some tests but I might not have time before next week.

I can probably fix the warning easily but it should not cause any issue
with the functionality.
